### DISCUSSION TITLE
unifying the 'step card' component pattern

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3361] [Q&A] [answered]
Hiya! Our goal is to create a unified `step card` component that can be utilized across multiple areas. To provide a brief explanation, a `step card` refers to cards with a step counter displayed on the left side, indicating the current step or stage. This design pattern is already prevalent in Segment.

At the moment, we have developed three different UI versions of the step card (ver. 1 isn't an actual card though, but context is the same), and our aim is to consolidate them into a single, cohesive version. We would greatly appreciate any suggestions or insights on which version we should proceed with. Thank you!

<img width="2505" alt="step-cards" src="https://github.com/twilio-labs/paste/assets/140410476/1a8819c4-9456-49c1-bbae-c30a19ed59ef">

**Comment by loreina** [comment, marked as answer]
Hi @marektonik,

I'd suggest moving forward with option 1 as that's directly from our Wizard layout which was designed as guidance across the Twilio product suite. It'd be great to use the same layout to establish consistency between products. cc @ceyerkes who can help with direction on the Segment side.

I've moved a ticket up in our backlog to polish the layout and create components from the visual explorations in Figma, and the team will follow up here when those components are ready. Thanks for your patience!
